plugin/evm/codec.go

```diff
@@ -21,6 +21,7 @@ func init() {
 
 	errs := wrappers.Errs{}
 	errs.Add(
+		c.RegisterType(&TestUnsignedTx{}),
```

Comment:
This is a breaking change since it will change the typeID used by import/export transactions.

Reply:
Ah this was not meant to be in the diff, this was for testing purposes. Will remove this.